WeatherApp/WeatherApp.Webpage/appsettings.json

```diff
@@ -6,5 +6,17 @@
       "Microsoft.Hosting.Lifetime": "Information"
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+  "ApiKeys": {
+    "Autocomplete": "jzxwrjVpz590MChJ0KjlrLnwg_syikNAPYB0tvSemLE",

```

Comment:
API keys are application secrets which should be in environment variable or user-secret instead. You can read more about the topic here: <https://docs.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-5.0&tabs=windows>
The configuration manager will take care of merge environment variables, user-secrets, appsettings.

Reply:
Thank you for your remark, it is fixed now.